src/gui/MainWindow.cpp

```diff
@@ -532,32 +531,6 @@ bool MainWindow::onKeyPressCallback(GtkWidget* widget, GdkEventKey* event, MainW
 		//editing text - give that control
 		return false;
 	}
-	else if (event->keyval == GDK_KEY_Down)
```

Comment:
Why is this removed?

Reply:
The same is done in XournalView.cpp:235 but this is called first.

Reply:
XournalView is the better position to do this. As that is actually the view that should react to the event.